pkg/services/ngalert/ngalert.go

```diff
@@ -312,13 +315,19 @@ func (ng *AlertNG) init() error {
 		statePersister = state.NewAsyncStatePersister(logger, ticker, cfg)
 	}
 	stateManager := state.NewManager(cfg, statePersister)
-	scheduler := schedule.NewScheduler(schedCfg, stateManager)
+	fetcherCfg := schedule.FetcherCfg{
+		ReloadInterval:       ng.Cfg.UnifiedAlerting.BaseInterval,
+		DisableGrafanaFolder: disableGrafanaFolder,
+	}
+	fetcher := schedule.NewBackgroundFetcher(fetcherCfg, ng.store, ng.Metrics.GetSchedulerMetrics(), log.New("ngalert.scheduler.fetcher"))
```

Comment:
Since we don't seem to use the `fetcher` or `scheduler` variables, should we just assign the fetcher and scheduler here?

```suggestion
	ng.fetcher := schedule.NewBackgroundFetcher(fetcherCfg, ng.store, ng.Metrics.GetSchedulerMetrics(), log.New("ngalert.scheduler.fetcher"))
```